numpy/__init__.cython-30.pxd

```diff
@@ -542,6 +542,12 @@ cdef extern from "numpy/arrayobject.h":
     void* PyArray_MultiIter_DATA(broadcast multi, npy_intp i) nogil
     void PyArray_MultiIter_NEXTi(broadcast multi, npy_intp i) nogil
     bint PyArray_MultiIter_NOTDONE(broadcast multi) nogil
+    npy_intp PyArray_MultiIter_SIZE(broadcast multi) nogil
+    int PyArray_MultiIter_NDIM(broadcast multi) nogil
+    npy_intp PyArray_MultiIter_INDEX(broadcast multi) nogil
+    int PyArray_MultiIter_NUMITER(broadcast multi) nogil
+    npy_intp* PyArray_MultiIter_DIMS(broadcast multi) nogil
+    void** PyArray_MultiIter_ITERS(broadcast multi) nogil
```

Comment:
I think we indeed need those, but we should also add `@property` using them so that you can keep using `bcast.size` and `bcast.shape` as you did before in cython.
(you can search the file for other properties, this is a new feature in Cython 3, that will be very useful to rely on for us in the future.)

Reply:
Added in 611f15deb81108f0846b630dc2e13d9d33f989e0